pipes/lucapipe.py

```diff
@@ -0,0 +1,18 @@
+from pipes.pipe import Pipe
+import numpy as np
+import cv2
+
+class LucaPipe(Pipe):
+    def pipe(self, data):
+        gray = cv2.cvtColor(data, cv2.COLOR_BGR2GRAY)
+        _, gray = cv2.threshold(gray, 128, 255, cv2.THRESH_BINARY)
+        kernel = np.ones((3,3),np.uint8)
+        gray = cv2.erode(gray, kernel, iterations = 1)
+        gray = cv2.dilate(gray, kernel, iterations = 1)
```

Comment:
Nice work! You should place a space behind the commas in line 9 and you should remove the spaces around the "=" in lines 10 & 11, to be conform with PEP8